components/Board/PlayArea/PlayArea.tsx

```diff
@@ -54,6 +56,7 @@ const PlayArea: FunctionComponent<Props> = ({}) => {
         <>
           <Feedback />
           {canSeeCard && <CardArea isHinter={isHinter} />}
+          {!canSeeCard && <Div text="You Are The Guesser" big />}
```

Comment:
This works. Another one as well would be using trinary `canSeeCard ? <CardArea/> : <Div/>`